### DISCUSSION TITLE
Usage with spyder

### DISCUSSION THREAD

**Discussion by daultimateboss85** [TomSchimansky/CustomTkinter#356] [General]
I recently installed anaconda... Spyder comes with it. I had customtkinter installed already and it still works in the that comes with python but in spyder it says no module name customtkinter. Is there something i'm missing or it doesn't work with spyder?

**Comment by TomSchimansky** [comment]
spyder ist just an IDE. If you can import customtkinter depends on if it is installed on the python version you are currently using. And with spyder you are probably using the anaconda python version. So you have to install customtkinter on the anaconda python version. In the terminal you have to activate your conda environment and then pip install customtkinter.

**Reply by daultimateboss85** [reply]
Thanks it worked,
